### PR TITLE
Add classes and tests from bibdata

### PR DESCRIPTION
**Pull request by hackartisan**

### CODE REVIEW

File: change_the_subject.gemspec [hunk 1]
```diff
@@ -26,9 +26,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_dependency "yaml"
 
-  # For more information and examples about making a new gem, checkout our
-  # guide at: https://bundler.io/guides/creating_gem.html
+  spec.add_development_dependency "pry-byebug"
```
Comment (maxkadel) [inline change_the_subject.gemspec:31]: If `lib/change_the_subject.rb` has `require "pry-byebug"`, if the application doesn't have pry-byebug already, won't it error, if here it's only in as a development dependency? (bibdata has it as a development & test dependency)

Comment (eliotjordan) [reply change_the_subject.gemspec:31]: Yep. Good catch.

Comment (eliotjordan) [reply change_the_subject.gemspec]: Fixed.

File: spec/change_the_subject_spec.rb [hunk 2]
```diff
@@ -1,11 +1,113 @@
...
+      expect(described_class.check_for_replacement(term: "Slaves")).to eq "Enslaved persons"
+      expect(described_class.check_for_replacement(term: "Women slaves")).to eq "Enslaved women"
+      expect(described_class.check_for_replacement(term: "Indians of Central America")).to eq("Indigenous peoples of Central America")
+      expect(described_class.check_for_replacement(term: "Indians of North America")).to eq("Indigenous peoples of North America")
+    end
   end
 
-  xit "does something useful" do
-    expect(false).to eq(true)
+  context "with a mocked configuration" do
```
[the remark this answers is not in the document]
Comment (eliotjordan) [reply spec/change_the_subject_spec.rb]: Fixed.

**Review by maxkadel — APPROVED** [review]
Looks great!

**Pull request merged**